### ISSUE TITLE
Style ark-ui components which are not available on park-ui

### ISSUE THREAD

**Issue by gurkerl83** [cschroeter/park-ui#82]
Is there a straightforward way to use the tailwind-plugin to style ark-components like DatePicker which is currently not available in park-ui?
Thx!

**Comment by cschroeter** [comment]
@sholzmayer any eta on this?

**Comment by sholzmayer** [comment]
The DatePicker is already integrated in tailwind on the Ark 1.0 PR, but it is not yet merged: https://github.com/cschroeter/park-ui/pull/65.

However the component code from panda is pretty similar and can just be copied over. Some modifications are needed:
- remove styled() wrapping around all components
- copy everything above the Root component from another ark-component
- adapt the ark imports

So it should looks somehow like
```ts
import * as Ark from "@ark-ui/react/date-picker";
import { createStyleContext } from "~/lib/create-style-context";
import { datePickerStyles } from "./recipe";
export * from "@ark-ui/react/date-picker";

const { withProvider, withContext } = createStyleContext(datePickerStyles);

const DatePickerRoot = withProvider(Ark.DatePicker.Root);
export const DatePickerClearTrigger = withContext(
  Ark.DatePicker.ClearTrigger,
  "clearTrigger"
);
export const DatePickerColumnHeader = withContext(
  Ark.DatePicker.ColumnHeader,
  "columnHeader"
);
export const DatePickerContent = withContext(Ark.DatePicker.Content, "content");
export const DatePickerControl = withContext(Ark.DatePicker.Control, "control");
export const DatePickerDayCell = withContext(Ark.DatePicker.DayCell);
export const DatePickerDayCellTrigger = withContext(
  Ark.DatePicker.DayCellTrigger,
  "cellTrigger"
);
export const DatePickerGrid = withContext(Ark.DatePicker.Grid, "grid");
export const DatePickerInput = withContext(Ark.DatePicker.Input, "input");
export const DatePickerMonthCell = withContext(Ark.DatePicker.MonthCell);
export const DatePickerMonthCellTrigger = withContext(
  Ark.DatePicker.MonthCellTrigger,
  "cellTrigger"
);
export const DatePickerMonthSelect = withContext(
  Ark.DatePicker.MonthSelect,
  "monthSelect"
);
export const DatePickerNextTrigger = withContext(
  Ark.DatePicker.NextTrigger,
  "nextTrigger"
);
export const DatePickerPositioner = withContext(
  Ark.DatePicker.Positioner,
  "positioner"
);
export const DatePickerPrevTrigger = withContext(
  Ark.DatePicker.PrevTrigger,
  "prevTrigger"
);
export const DatePickerRow = withContext(Ark.DatePicker.Row, "row");
export const DatePickerRowGroup = withContext(
  Ark.DatePicker.RowGroup,
  "rowGroup"
);
export const DatePickerRowHeader = withContext(
  Ark.DatePicker.RowHeader,
  "rowHeader"
);
export const DatePickerTrigger = withContext(Ark.DatePicker.Trigger, "trigger");
export const DatePickerViewTrigger = withContext(
  Ark.DatePicker.ViewTrigger,
  "viewTrigger"
);
export const DatePickerYearCell = withContext(Ark.DatePicker.YearCell);
export const DatePickerYearCellTrigger = withContext(
  Ark.DatePicker.YearCellTrigger,
  "cellTrigger"
);
export const DatePickerYearSelect = withContext(
  Ark.DatePicker.YearSelect,
  "yearSelect"
);

export const DatePicker = Object.assign(DatePickerRoot, {
  Root: DatePickerRoot,
  ClearTrigger: DatePickerClearTrigger,
  ColumnHeader: DatePickerColumnHeader,
  Content: DatePickerContent,
  Control: DatePickerControl,
  DayCell: DatePickerDayCell,
  DayCellTrigger: DatePickerDayCellTrigger,
  Grid: DatePickerGrid,
  Input: DatePickerInput,
  MonthCell: DatePickerMonthCell,
  MonthCellTrigger: DatePickerMonthCellTrigger,
  MonthSelect: DatePickerMonthSelect,
  NextTrigger: DatePickerNextTrigger,
  Positioner: DatePickerPositioner,
  PrevTrigger: DatePickerPrevTrigger,
  Row: DatePickerRow,
  RowGroup: DatePickerRowGroup,
  RowHeader: DatePickerRowHeader,
  Trigger: DatePickerTrigger,
  ViewTrigger: DatePickerViewTrigger,
  YearCell: DatePickerYearCell,
  YearCellTrigger: DatePickerYearCellTrigger,
  YearSelect: DatePickerYearSelect,
});

``` 


And the recipe.ts has a slot <> className mapping with all used class names in the component.

```ts
import { tv } from "tailwind-variants";

export const datePickerStyles = tv({
  "base": "datePicker",
  "slots": {
    "cellTrigger": "datePicker__cellTrigger",
    "clearTrigger": "datePicker__clearTrigger",
    "content": "datePicker__content",
    "control": "datePicker__control",
    "grid": "datePicker__grid",
    "input": "datePicker__input",
    "monthSelect": "datePicker__monthSelect",
    "nextTrigger": "datePicker__nextTrigger",
    "positioner": "datePicker__positioner",
    "prevTrigger": "datePicker__prevTrigger",
    "rowHeader": "datePicker__rowHeader",
    "trigger": "datePicker__trigger",
    "viewTrigger": "datePicker__viewTrigger",
    "yearSelect": "datePicker__yearSelect",
    "row": "datePicker__row",
    "rowGroup": "datePicker__rowGroup",
    "columnHeader": "datePicker__columnHeader"
  },
  "variants": {}
});
```

The demo snippet can be copied. Just make sure there are no styled-system properties like `gap="3"` on the components. These need to be replaced with tailwind classes instead.

Not sure if that works out of the box for you, but it is at least a starting point.

**Comment by sholzmayer** [comment]
@cschroeter regarding the ETA on this: it will be available with the ark 1.0
